server/models/userEvent.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose, { Document, Schema } from "mongoose";
+import { UserEventRequest } from "../types/requests/userEventRequest";
+
+const eventSchema = new Schema<UserEventRequest & Document>({
+	name: {
+		type: String,
+		required: true
+	},
+	userid: {
+		type: Schema.Types.ObjectId,
+		required: true,
+		ref: "User"
+	},
+	day: {
+		type: Number,
+		required: true
+	},
+	month: {
+		type: Number,
+		required: true
+	},
+	year: {
+		type: Number,
+		required: true
+	},
+	start: {
```

Comment:
24 hour time right? so then how would I say I want to start something at 12:27AM? woul I say 27 because I cant say 027. Ig backend will worry about that.